### ISSUE TITLE
[New Rule] Use any / all

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#15]
## Explanation

It's shorter.

## Example 1: any

```python
# Bad
for x in iterable:
    if check(x):
        return True
return False

# Good
return any(check(x) for x in iterable)
```

## Example 2: all

```python
# Bad
for x in iterable:
    if check(x):
        return False
return True

# Good
return all(check(x) for x in iterable)
```
